connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorPluginsResource.java

```diff
@@ -48,7 +50,11 @@ public ConnectorPluginsResource(Herder herder) {
     @Path("/{connectorType}/config/validate")
```

Comment:
It's definitely tempting. I guess API change means a KIP though, so maybe we should just put this in the backlog as a potential cleanup for the future.